### DISCUSSION TITLE
Question: How to represent a face position on a 2D canvas?

### DISCUSSION THREAD

**Discussion by nellfs** [luicfrr/react-native-vision-camera-face-detector#63] [Q&A]
I need to get a `Face` `x` and `y` position to check if the face is centered on a 2D shape (like a square), is there any way of doing so?

I already tried `bounds` and `landmarks`, but the position does not follow the center of my face when I get a little far or a little closer to the camera... Is there any way to use detection in just a point of the camera?

**Comment by luicfrr** [comment]
You can use face bounds to determine the center of the face. Then you need to use frame sizes to determine the horizontal/vertical center of the screen and add/subtract from these values the range you want the face to be.

In my local testa disabling `autoscale` feature makes this easer because of devices screen scale.
